### PR TITLE
Latest docs links

### PR DESCRIPTION
**Pull request by dkillick**
Adds a link to the latest versions of the Iris and Cartopy docs to the homepage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,11 +42,25 @@
           <div class="projects">
             <div class="project">
                 <div class="logo"><a href="iris/index.html"><img src="images/Iris7_1_trim_100.png" style="height:100;" alt="Iris Logo" /></a></div>
-                <div class="summary"><a href="iris/index.html">Iris</a> is a Python package for analysing and visualising meteorological and oceanographic data sets.</div>
+                <div class="summary">
+                    <p>
```
Comment (dkillick) [inline index.html]: We had naked text (not encapsulated in a paragraph tag) before.

File: index.html [hunk 2]
```diff
@@ -42,11 +42,25 @@
...
+                        <br />
+                        See the <a href="iris/docs/latest/index.html">latest documentation</a> for Iris.
+                    </p>
+                </div>
             </div>
             <div class="project">
                 <div class="logo"><a href="cartopy/index.html"><img src="images/cartopy_small.png" style="height:100;" alt="Cartopy logo" /></a></div>
-                <div class="summary"><a href="cartopy/index.html">Cartopy</a> is a Python package for advanced map generation with a simple matplotlib interface.</div>
+                <div class="summary">
+                    <p>
```
Comment (dkillick) [inline index.html]: Fixed more naked text

**Comment by ajdawson** [comment]
I think you are supposed to edit `source/index.html` then run `build.py` to produce this file.

**Comment by dkillick** [comment]
You're right, I am... I am fixing that literally now!

**Comment by dkillick** [comment]
Rebased to remove conflicts.

**Pull request merged**